backend/lib/cache/index.js

```diff
@@ -15,29 +15,58 @@
 //
 
 const _ = require('lodash')
+const { HTTPError } = require('got')
+const { Store } = require('../kubernetes-client/cache')
+const createJournalCache = require('./journals')
 
-const cloudProfiles = []
-const seeds = []
-const quotas = []
-const journals = require('./journals')()
+class Cache {
+  constructor () {
+    this.synchronizationTriggered = false
+    this.cloudprofiles = new Store()
+    this.seeds = new Store()
+    this.quotas = new Store()
+    this.projects = new Store()
+    this.journalCache = createJournalCache()
+  }
+
+  synchronize (client) {
+    if (!this.synchronizationTriggered) {
+      client['core.gardener.cloud'].cloudprofiles.syncList(this.cloudprofiles)
+      client['core.gardener.cloud'].quotas.syncListAllNamespaces(this.quotas)
+      client['core.gardener.cloud'].seeds.syncList(this.seeds)
+      client['core.gardener.cloud'].projects.syncList(this.projects)
+      this.synchronizationTriggered = true
+    }
+  }
 
-const cache = {
   getCloudProfiles () {
-    return cloudProfiles
```

Comment:
why do we sort all the cached values? is this required?

Reply:
Sometimes it is required. The cache should behave consistently.

Reply:
we agreed to remove sorting in the cache